frontend/src/components/Tabs/AdminTabs/RoleRequest.js

```diff
@@ -15,128 +15,142 @@ function RoleRequest(props) {
   const [immigrationOfficialList, setIOList] = useState([]);
 
 
-  useEffect(async () => {
+  useEffect(() => {
     console.log("Patient List is Loaded!");
-    await axios.get(`admin/${currentUser.user.uid}/pending-patients`)
-        .then((response) => {
-          setPatientList(response.data.users);
-          console.log(response.data);
-        })
-        .catch((error) => {
-          console.log(error.response);
-        })
-
-    await axios.get(`admin/${currentUser.user.uid}/pending-doctors`)
-        .then((response) => {
-          setDoctorList(response.data.doctors);
-          console.log(response.data);
-        })
-        .catch((error) => {
-          console.log(error.response);
-        })
-
-    await axios.get(`admin/${currentUser.user.uid}/pending-health-officials`)
-        .then((response) => {
-          setHOList(response.data.healthOfficials);
-          console.log(response.data);
-        })
-        .catch((error) => {
-          console.log(error.response);
-        })
-
-    await axios.get(`admin/${currentUser.user.uid}/pending-immigration-officials`)
-        .then((response) => {
-          setIOList(response.data.immigrationOfficials);
-          console.log(response.data);
-        })
-        .catch((error) => {
-          console.log(error.response);
-        })
-    }, []);
-
-  function renderPendingPatientList() {
-    if (patientList === null) return;
+    axios.get(`admin/${currentUser.user.uid}/pending-patients`)
+      .then((response) => {
+        setPatientList(response.data.users);
+        console.log(response.data);
+      })
+      .catch((error) => {
+        console.log(error.response);
+      })
+
+    axios.get(`admin/${currentUser.user.uid}/pending-doctors`)
+      .then((response) => {
+        setDoctorList(response.data.doctors);
+        console.log(response.data);
+      })
+      .catch((error) => {
+        console.log(error.response);
+      })
+
+    axios.get(`admin/${currentUser.user.uid}/pending-health-officials`)
+      .then((response) => {
+        setHOList(response.data.healthOfficials);
+        console.log(response.data);
+      })
+      .catch((error) => {
+        console.log(error.response);
+      })
+
+    axios.get(`admin/${currentUser.user.uid}/pending-immigration-officials`)
+      .then((response) => {
+        setIOList(response.data.immigrationOfficials);
+        console.log(response.data);
+      })
+      .catch((error) => {
+        console.log(error.response);
+      })
+  }, []);
+
+  // function renderPendingPatientList() {
+  //   if (patientList === null) return;
+  //   return <div>
+  //     {patientList.map((patient, index) => <RoleRequestBox
+  //       key={index}
+  //       RequesterUsername={patient.name}
+  //       RequestType={patient.userType}
```

Comment:
Please remove these commented lines. Commented code should not appear in a pull request